components/dashboard/src/projects/ConfigureProject.tsx

```diff
@@ -93,52 +125,121 @@ export default function () {
-        <h3 className="text-center">Output</h3>
-        {!!workspaceCreationResult && <PrebuildLogs workspaceId={workspaceCreationResult.createdWorkspaceId} />}
+      <div className="flex-grow h-96 rounded-xl overflow-hidden bg-gray-100 dark:bg-gray-700 flex flex-col">
+        <div className="flex-grow flex">{workspaceCreationResult
+          ? <PrebuildLogs workspaceId={workspaceCreationResult.createdWorkspaceId} onInstanceUpdate={onInstanceUpdate} />
+          : <div className="flex-grow flex flex-col items-center justify-center">
+              <img className="w-14" role="presentation" src={isDark ? PrebuildLogsEmptyDark : PrebuildLogsEmpty} />
+              <h3 className="text-center text-lg text-gray-500 dark:text-gray-50 mt-4">No Recent Prebuild</h3>
+              <p className="text-center text-base text-gray-500 dark:text-gray-400 mt-2 w-64">Edit the project configuration on the left to get started. <a className="learn-more" href="https://www.gitpod.io/docs/config-gitpod-file/">Learn more</a>.</p>
+            </div>
+        }</div>
+        <div className="h-20 px-6 bg-gray-50 dark:bg-gray-800 border-t border-gray-200 dark:border-gray-600 flex space-x-2">
+          {prebuildWasTriggered && <PrebuildStatus prebuildPhase={prebuildPhase} isDark={isDark} />}
+          <div className="flex-grow" />
+          <button className="secondary">New Workspace</button>
+          <button disabled={isEditorDisabled} onClick={buildProject}>Run Prebuild</button>
+        </div>
       </div>
     </div>
+  </>;
+}
+
+function EditorMessage(props: { heading: string, message: string, type: 'success' | 'warning' }) {
+  return <div className={`p-4 flex flex-col ${props.type === 'success' ? 'bg-green-50 text-green-700' : 'bg-yellow-50 text-yellow-700'}`}>
+    <strong className={`font-semibold ${props.type === 'success' ? 'text-green-800' : 'text-yellow-800'}`}>{props.heading}</strong>
+    <span>{props.message}</span>
+  </div>;
+}
+
+function PrebuildStatus(props: { prebuildPhase?: string, isDark?: boolean }) {
+  let status = <></>;
+  let details = <></>;
+  switch (props.prebuildPhase) {
+    case undefined: // Fall through
+    case 'unknown':
+      status = <div className="flex space-x-1 items-center text-yellow-600">
+        <img className="h-4 w-4" src={StatusPaused} />
+        <span>PENDING</span>
+      </div>;
+      details = <div className="flex space-x-1 items-center text-gray-400">
+        <img className="h-4 w-4 animate-spin" src={props.isDark ? SpinnerDark : Spinner} />
+        <span>Prebuild in progress</span>
+      </div>;
+      break;
+    case 'preparing': // Fall through
+    case 'pending': // Fall through
+    case 'creating': // Fall through
+    case 'initializing': // Fall  through
+    case 'running': // Fall through
+    case 'interrupted':
+      status = <div className="flex space-x-1 items-center text-blue-600">
+        <img className="h-4 w-4" src={StatusRunning} />
+        <span>RUNNING</span>
```

Comment:
**issue**: Prebuilds run in the configuration page do not surface in the prebuilds page. I'd think all triggered prebuilds here would end up in the prebulds page. What do you think? Cc @AlexTugarev @svenefftinge

Reply:
I think they should, and it could be a bug that they don't (the configurator currently saves the project configuration and then triggers a regular prebuild for the default branch)